core/src/main/scala/pureconfig/BasicConverters.scala

```diff
@@ -0,0 +1,106 @@
+package pureconfig
+
+import java.net.{ URI, URL }
+import java.nio.file.{ Path, Paths }
+import java.time._
+import java.util.UUID
+
+import scala.concurrent.duration.{ Duration, FiniteDuration }
+
+import com.typesafe.config._
+import pureconfig.error._
+
+/**
+ * Trait containing `ConfigConvert` instances for primitive types and simple classes in Java and Scala standard
+ * libraries.
+ */
+trait BasicConverters extends ConvertHelpers {
+
+  // primitive type converters
```

Comment:
If you'll permit the logical fallacy of appeal to authority, I'll refer you to how Cats did something similar to this here:
https://github.com/typelevel/cats/blob/master/core/src/main/scala/cats/instances/anyval.scala#L4